src/main/resources/database/changes/release-10.60.0/add_no_active_enrolments_category.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO casesvc.category ("categorypk", "shortdescription", "longdescription", "eventtype", "role", "generatedactiontype", "group", "oldcasesampleunittypes", "newcasesampleunittype", "recalccollectioninstrument")
+VALUES ('NO_ACTIVE_ENROLMENTS', 'No active enrolments', 'No active enrolments remaining for case', 'ACTIVATED', NULL, NULL, NULL, 'B', NULL, NULL)
```

Comment:
You should probably add an integration test around this.

Reply:
Looking into this now 👍

Reply:
Had a look into this. If we want to do an integration test for posting the `NO_ACTIVE_ENROLMENTS` category we're going to have to handle calls to `iac` and `action` services. We could spin these up in docker or mock them but It will probably take some time as there's no clear way of doing this in the current integration tests I don't think

Reply:
So might leave this out for this test, a lot of this process will be changing a lot with the rest of the `BI` case work anyway

Reply:
Umm why is it `oldcasesampleunittypes` and `newcasesampleunittype` and not either type/types for both for consistency?